### PR TITLE
fix: change version constraint of laminas-skeleton-installer

### LINKED ISSUE #28: skeleton-installer 0.1.7 not compatible with composer 2
### Bug Report

|    Q        |   A
|------------ | ------
| Version(s)  | 1.0.0

#### Summary

This package currently requires version `0.1.7` of `laminas/laminas-skeleton-installer` (or not yet released version `1.0`) which is not compatible with composer v2.

#### Current behavior

As noted by a [user on laminas slack](https://laminas.slack.com/archives/C4QBQUEG5/p1603730101252400) creating a project with this skeleton fails when using composer 2.

#### Expected behavior

Adding `^0.2` to the `laminas/laminas-skeleton-installer` constraint would probably resolve the error (I have not tested it, yet).

### PR DESCRIPTION
**Pull request by rieschl**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | yes
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

Add `^0.2` to the version constraint of `laminas/laminas-skeleton-installer`.
Version `^0.2` already supports composer 2.

I tried it locally with composer `1.10.16` and `2.0.2` without problems.

fixes #28

### CODE REVIEW

**Review by samsonasik — APPROVED** [review]

**Comment by TomHAnderson** [comment]
👍

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         "php": "^7.3",
         "laminas/laminas-component-installer": "^1.0 || ^2.1",
         "laminas/laminas-development-mode": "^3.2",
-        "laminas/laminas-skeleton-installer": "^0.1.7 || ^1.0",
+        "laminas/laminas-skeleton-installer": "^0.1.7 || ^0.2 ||  ^1.0",
```
Comment (Xerkus) [inline composer.json]: Please drop  0.1.7 from dependencies, We need only latest version here,

**Review by Xerkus — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (rieschl) [reply composer.json]: done. but I think changing a major version of a constraint requires a minor version change, not just a patch release...
So I targeted the wrong branch.

**Comment by samsonasik** [comment]
@rieschl I changed the target branch.

**Comment by samsonasik** [comment]
@rieschl merged to 1.1.x branch, thank you.

**Comment by rieschl** [comment]
> 
> 
> @rieschl I changed the target branch.

thanks, @samsonasik for your help!

**Pull request merged**